sagemaker_batch_transform/custom_tensorflow_inference_script_csv_and_tfrecord/custom_tensorflow_inference_script_csv_and_tfrecord.ipynb

```diff
@@ -0,0 +1,717 @@
+{
```

Comment:
<p>"Here, we train the network using the TensorFlow .fit method, just like if we were using our local computers." -&gt; "Here, we train the network using the TensorFlow .fit method."</p><p> </p><p>Not exactly right to say it is on our local machines - more like it is on SageMaker machines. Best to omit together</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3438/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Done